src/main/target/STM32_UNIFIED/target.h

```diff
@@ -211,9 +211,9 @@
 #define USE_TRANSPONDER
 
 //TODO: Make this actually work by making the pins configurable
-#define USE_RANGEFINDER
```

Comment:
Is there any reason to remove this from STM32F405? Saving 1248 bytes on a target that has ~ 500 kB of unused flash space seems to be moot, and if we stop building this then it will rot and become unusable in no time at all, so we can just as well delete the code.

Reply:
Well as noted in the comment it doesn't work for unified targets anyway.